app/controllers/players_controller.rb

```diff
@@ -26,28 +28,34 @@ def edit
   def destroy
     @player = Player.find(params[:id])
     @player.destroy
-    respond_to do |format|
-      format.html { redirect_to players_url, notice: 'Post was successfully destroyed.' }
-      format.json { head :no_content }
-    end
+    redirect_to players_url, notice: 'Post was successfully destroyed.'
   end
 
   def update
     @player = Player.find(params[:id])
-    respond_to do |format|
-      if @player.update(player_params)
-        format.html { redirect_to players_url, notice: 'Post was successfully updated.' }
-        format.json { render :show, status: :ok, location: @post }
-      else
-        format.html { render :edit }
-        format.json { render json: @player.errors, status: :unprocessable_entity }
-      end
+    yellow = params[:player][:yellow_cards]
+    # rubocop: disable Rails/SkipsModelValidations
```

Comment:
instead of skipping rubocop validation warning, use a method that would trigger them, like `update`